signer/apk2/apk2.go

```diff
@@ -0,0 +1,148 @@
+package apk2
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"crypto/ecdsa"
+	"crypto/sha256"
+	"crypto/x509"
+	"os"
+	"os/exec"
+
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+	"go.mozilla.org/autograph/signer"
+)
+
+const (
+	// Type of this signer is "apk2" represents a signer that
+	// shells out to apksigner to sign artifacts
+	Type = "apk2"
+)
+
+// APK2Signer holds the configuration of the signer
+type APK2Signer struct {
+	signer.Configuration
+	minSdkVersion string
```

Comment:
Is this the installed apksigner SDK version or the minimum client SDK version the signature should support?

Reply:
It's the minimum client sdk. I'll add a comment.